ui/charsettings.py

```diff
@@ -205,6 +220,13 @@ async def get_content(self):
             f"*If this is enabled, your coins will be displayed in decimal gold format.*",
             inline=False,
         )
+        embed.add_field(
+            name="Coin Conversion Mode",
+            value=f"**{autoconvert_coins_desc(self.settings.autoconvert_coins)}**\n"
+            f"*Whether or not it should ask you if you would like to auto-convert your coins if you try so spend "
+            f"more than you currently have.*",
```

Comment:
```suggestion
            f"*If a coin transaction would cause you to have a negative number of a certain currency, whether to automatically convert other coins to cover the transaction.*",
```